src/extension.ts

```diff
@@ -1282,6 +1283,16 @@ async function createClusterKubernetes(request? : WizardUIRequest) {
     }
 }
 
+async function useContextKubernetes(explorerNode: explorer.ResourceNode) {
+    const targetContext = explorerNode.metadata.context;
+    const shellResult = await kubectl.invokeAsync(`config use-context ${targetContext}`);
+    if (!shellResult.code) {
```

Comment:
Inverted condition makes this hard to read.  Consider writing `if (shellResult.code === 0)`.

Reply:
nice.